src/calculateRentalCost.js

```diff
@@ -29,6 +29,13 @@
 
 function calculateRentalCost(days) {
   // write code here
+  if (days >= 7) {
+    return (days * 40) - 50;
+  } else if (days >= 3) {
+    return (days * 40) - 20;
+  } else {
+    return (days * 40);
+  }
```

Comment:
There are a lot of magic numbers here. It's a bad practice, so read carefully the checklist to create constants with meaningful names https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md